src/ucp/core/ucp_ep.c

```diff
@@ -647,7 +648,7 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
                     config->am.zcopy_thresh[0]      = ucs_max(context->config.ext.zcopy_thresh,
                                                               iface_attr->cap.am.min_zcopy);
                 }
-
+                config->am.zcopy_enabled = 1;
```

Comment:
as we discussed, this could be added to line 638 instead, and renamed to 'config->am.zcopy_auto_thresh' - to avoid checking 'worker->context->config.ext.zcopy_thresh != UCS_CONFIG_MEMUNITS_AUTO' in tag send flow